### PR TITLE
Replace jupiter by hamcrest

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

Fixes: https://github.com/strimzi/strimzi-kafka-bridge/issues/352

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -154,6 +134,12 @@
 			<version>2.12.4</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-all</artifactId>
+			<version>1.3</version>
```
Comment (ppatierno) [inline pom.xml]: avoid to use fixed version here, define a property instead as for other dependencies.

**Comment by stanlyDoge** [comment]
It should be because of better failure messages: https://objectpartners.com/2013/09/18/the-benefits-of-using-assertthat-over-other-assert-methods-in-unit-tests/

**Comment by ppatierno** [comment]
About failure messages, I agree but saying that `assertEquals(expected, actual);` is " very cryptic" ... :-D

**Comment by ppatierno** [comment]
@Seequick1  @Frawless any more comments?

**Comment by Seequick1** [comment]
Everything seems fine for me. LGTM

**Pull request merged**